clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/images/memory/FileMemoryAccessObject.kt

```diff
@@ -1,12 +1,14 @@
 package com.clevertap.android.sdk.inapp.images.memory
 
+import com.clevertap.android.sdk.ILogger
 import com.clevertap.android.sdk.inapp.images.memory.MemoryDataTransformationType.ToBitmap
 import com.clevertap.android.sdk.inapp.images.memory.MemoryDataTransformationType.ToByteArray
 import com.clevertap.android.sdk.inapp.images.memory.MemoryDataTransformationType.ToFile
+import com.clevertap.android.sdk.inapp.images.repo.TAG_FILE_DOWNLOAD
 import com.clevertap.android.sdk.utils.CTCaches
 import java.io.File
 
-internal class FileMemoryAccessObject(private val ctCaches: CTCaches): MemoryAccessObject<ByteArray> {
+internal class FileMemoryAccessObject(private val ctCaches: CTCaches,private val logger: ILogger?): MemoryAccessObject<ByteArray> {
```

Comment:
lets pass null default for logger

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/2dff9d315abf71c5a15557096c27aa731db5e33a